_docs/workflows.md

```diff
@@ -64,19 +64,23 @@ Now that you have linked your credentials and refreshed, there should be one stu
 
-Just like with Tools, a workflow is not visible on the public 'Workflows' listing unless it is published. To publish a container, click the yellow 'Publish' button in the top-right corner.
+Just like with Tools, a workflow is not visible on the public 'Workflows' listing unless it is published. To publish a container, click the blue 'Publish' button in the top-right corner.
 
 #### Manual Registration of Workflows
 
 In certain cases, you may wish to register workflows in a different source code structure, especially when working with complex project structures. For example, if you want to register two workflows from the same repository.
 
-Workflows can be registered manually from the 'My Workflows' page by pressing the 'Register Workflow' button at the bottom of the right side bar, or any of the '+' buttons in each accordion namespace group. A modal will appear as below:
+Workflows can be registered manually from the 'My Workflows' page by pressing the 'Register Workflow' button at the bottom of the right side bar. A modal will appear as below:
 
 ![Register Workflow Manual](/assets/images/docs/register_workflow_manual.png)
 
-The Source Code Repository field must be filled out and is in the format `namespace/name` (the two paths may differ). The Workflow (descriptor) path and test parameter path are relative to the root of the Source Code Repository (and must begin with '/'), these will be the default locations to find their corresponding files, unless specified otherwise in the tags. The toolname is an optional 'suffix' appended to the Dockstore path, it allows for two workflows to share the same Git paths; the toolname uniquely distinguishes image repositories in Dockstore.
+The first option allows you to register a workflow from an existing third party repository.  The second option allows you to write your workflow files and store them directly on Dockstore.org. You can read more about hosted workflows [here](/docs/publisher-tutorials/hosted-tools-and-workflows/), but for now, let's select the first option.  Clicking 'Next' will bring up the following modal: 
+
+![Register Workflow Manual2](/assets/images/docs/register_workflow_manual2.png)
+
+Source Code Provider allows you to choose between GitHub, BitBucket, and GitLab (your respective accounts for these third party repositories need to be linked to your Dockstore account). The Source Code Repository field must be filled out and is in the format `namespace/name` (the two paths may differ). The Workflow (descriptor) path and test parameter path are relative to the root of the Source Code Repository (and must begin with '/'). These will be the default locations to find their corresponding files, unless specified otherwise in the tags. The Workflow Name is an optional 'suffix' appended to the Dockstore path. It allows for two workflows to share the same Git paths; the Workflow Name uniquely distinguishes workflow repositories in Dockstore.
```

Comment:
Not within the scope of the ticket, but @garyluu @agduncan94 
Is there a reason why the "Add tag" button and "close" are left aligned in assets/images/docs/tageditor_modal.png now whereas  the button is on the right for "Register Tool", "Register Workflow", "Create Organization Request", etc.?

Reply:
just forgotten probably